### DISCUSSION TITLE
PyPI latest version is still 0.0.32

### DISCUSSION THREAD

**Discussion by gbronca** [Aleksi44/wagtailsvg#19] [General]
Hi,

I've noticed that PyPI still shows the latest version as v0.0.32 instead of 0.0.33, which is compatible with Wagtail 3.

Any idea when PyPI will be updated?

Thanks

**Comment by Aleksi44** [comment]
Hi @gbronca , I just updated the pypi package

**Reply by gbronca** [reply]
That's great! Thanks
